### ISSUE TITLE
Impossible to lock voice note with Q fully gestural nav

### ISSUE THREAD

**Issue by u32i64** [signalapp/Signal-Android#8918]
### Bug description
It is impossible to "lock" voice note recording when the Android Q fully gestural navigation is used unless the keyboard is visible.

### Steps to reproduce
- Select conversation.
- Make sure keyboard is hidden.
- Hold "record voice note" button.
- Slide up as prompted to "lock" the recording.

**Actual result:** Android starts going to the Home screen **and** the unfinished voice note is sent by Signal if its length reaches the "anti-tap" threshold.
**Expected result:** The voice note recording is "locked".

### Screen capture
[![1]][1]

[1]: https://user-images.githubusercontent.com/19972511/61067864-4de89c00-a411-11e9-8034-953a19c0eace.gif

### Device info
**Device:** Pixel 3 XL
**Android version:** Q Beta 4
**Signal version:** 4.43.8

**Comment by alan-signal** [comment]
Can you tell me if there is also any issue dragging things inside the image editor?

**Comment by u32i64** [comment]
**TL;DR**: Yes

- If the "sticker" is too low, it's possible to pick it up, however it does move only slightly before the navigation kicks in. This applies both to the "edit or delete this sticker" mode as well as to the top-level "general editing" mode.
- Pen/highlighter are tricky to use just as well near the edge.

All of the above only apply if the drag was **started** "close enough" to the bottom of the screen.

- Text looks like not draggable at all in the part of the screen covered by the message box and thumbnail area, I'm not sure if this is intended or not.
- Crop tool seems to be very smart and ignores all of my attempts to get the handle low enough so I can start the drag there, but I assume if you can get the handles there the issue would be present as well.

**Comment by greyson-signal** [comment]
I think the release version of Q changed some stuff, because I can no longer repro.

**Comment by u32i64** [comment]
@greyson-signal Quite reproducible on:

**Device**: Pixel 3 XL
**Android version**: 10 
**Signal version**: 4.49.18

![](https://user-images.githubusercontent.com/19972511/68528812-b33d9280-0308-11ea-9240-c760708ce45c.gif)
